src/Shapes/setCollisionSmoothingRadius.jl

```diff
@@ -30,3 +30,37 @@ function setCollisionSmoothingRadius(shape::Beam{F}, collisionSmoothingRadius::F
     collisionSmoothingRadius2 = min(collisionSmoothingRadius, F(0.1)*min(shape.length, shape.width, shape.thickness)) # at most 10% of the smallest edge length
     return collisionSmoothingRadius2
 end
+
+
+setContactSphereRadius(shape::Nothing, contactSphereRadius::Nothing, ::Type{F}) where F <: Modia3D.VarFloatType = (false, F(NaN))
+
+setContactSphereRadius(shape::Sphere{F}, contactSphereRadius::Nothing, ::Type{F}) where F <: Modia3D.VarFloatType = (false, F(shape.diameter * 0.5) )
+setContactSphereRadius(shape::Sphere{F}, contactSphereRadius::F, ::Type{F}) where F <: Modia3D.VarFloatType = (false, contactSphereRadius)
+
+
+setContactSphereRadius(shape::Ellipsoid{F}, contactSphereRadius::Nothing, ::Type{F}) where F <: Modia3D.VarFloatType = (false, min(shape.lengthX, shape.lengthY, shape.lengthZ)*F(0.5) )
+setContactSphereRadius(shape::Ellipsoid{F}, contactSphereRadius::F, ::Type{F}) where F <: Modia3D.VarFloatType = (false, contactSphereRadius )
+
+
+setContactSphereRadius(shape::Cylinder{F}, contactSphereRadius::Nothing, ::Type{F}) where F <: Modia3D.VarFloatType = (false, min(shape.diameter, shape.length)*F(0.5) )
```

Comment:
Isn't F() required here?

Reply:
I changed it, it's consistent with the other functions now.